### PR TITLE
Fix #82199, resetting some color of code blocks at Markdown Preview with light theme

### LINKED ISSUE #82199: Markdown: color of numbers in code blocks is not suitable with light theme at Preview
In Markdown preview with light theme, numbers in code blocks are not clearly distinguishable because of their color. It will be good if a different color can be switched in light theme mode

image

It seems that in `/extensions/markdown-language-features/media/highlight.css` , the following part may add a `.vscode-light` switch. Thank you!

 ```css
.hljs-number,
.hljs-class {
	color: #B8D7A3;
}
```

### PR DESCRIPTION
**Pull request by ghost**
This PR fixes #82199, which is opened by myself, that resets color theme of number, title and built_in in code blocks at Preview with light theme, so that they can be easier to read.

**Resetting Before**
before

**Resetting After**
fixe

### CODE REVIEW

File: extensions/markdown-language-features/media/highlight.css [hunk 1]
```diff
@@ -121,6 +121,11 @@ From https://raw.githubusercontent.com/isagalaev/highlight.js/master/src/styles/
 Visual Studio-like style based on original C# coloring by Jason Diamond <jason@diamond.name>
 
 */
+/*
```
Comment (mjbvz) [inline extensions/markdown-language-features/media/highlight.css]: No owner comments please

**Review by mjbvz — CHANGES_REQUESTED** [review]
Change looks good overall, just remove the owner comment. Thanks

I already merged #82450 since it came in first and fixed the problem in a scoped way

File: extensions/markdown-language-features/media/highlight.css (hunk 1 above)
Comment (ganxy1028) [reply extensions/markdown-language-features/media/highlight.css]: Remove already

Comment (ganxy1028) [reply extensions/markdown-language-features/media/highlight.css]: Remove done

**Comment by mjbvz** [comment]
Thanks!

**Pull request merged**